### DISCUSSION TITLE
Extended battery?

### DISCUSSION THREAD

**Discussion by richardw347** [iRobotEducation/create3_docs#383] [Questions] [answered]
### How are you connecting to your Create 3?

USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Raspberry Pi4 / Ubuntu

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.1

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

There are larger capacity batteries available for previous version of the Create (also the commercial Roombas). I was just wondering if there were plans to make one available for the Create 3? 

I'm working on a project which adds a few extra sensors so it would be nice to have a bit more than the 1800 mAh battery that it has currently and there seems to be space for a higher capacity battery.

**Comment by shamlian** [comment, marked as answer]
Hello!

The [battery for all j-series and some i-series Roomba robots](https://www.irobot.com/en_US/lithium-ion-battery-for-roomba/4706313.html) will work with Create 3, and is ~32 Wh (compared to the ~26 Wh of the [original battery](https://www.irobot.com/en_US/roomba-accessories/4624864.html)), as long as your robot's software is up-to-date (and I can see that it is). The battery for the Roomba Combo™ j7+ should also work with the robot, and is ~64 Wh. Unfortunately, that battery is not yet available separately, but I suspect it will be, some time soon.

Hope this helps!

**Reply by richardw347** [reply]
That does answer the question, thank you!
